integration_tests/dbt_project.yml

```diff
@@ -4,30 +4,25 @@ profile: 'integration_tests'
 config-version: 2
 
 vars: 
+  google_play_schema: google_play_source_integration_tests
```

Comment:
These changes are intended to make the experience of navigating our generated dbt docs easier for the user. With the identifier being leveraged instead of the base variable, we are able to remove any floating seed files. This results in a more natural looking environment.

Reply:
ah right! sounds good, i will regenerate the docs